.github/workflows/main.yml

```diff
@@ -0,0 +1,86 @@
+name: Build and Test
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
```

Comment:
Shouldn't it monitor also the openSUSE branch itself?

Reply:
Good point!